apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/util/CommonlyUsedPwdCheckerTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.portal.util;
+
+import java.util.Arrays;
+import java.util.List;
+import org.apache.commons.lang.RandomStringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CommonlyUsedPwdCheckerTest {
+
+  @Test
+  public void testMatch() {
+    List<String> commonlyUsedPwdList = Arrays.asList(
+        "12345678", "98765432", "11111111", "22222222", "33333333", "44444444",
+        "55555555", "66666666", "77777777", "88888888", "99999999", "00000000",
+        "1q2w3e4r", "qwertyuiop", "asdfghjkl", "asdfghjkl", "abcd1234"
+    );
+
+    for (String pwd : commonlyUsedPwdList) {
+      Assert.assertTrue(CommonlyUsedPwdChecker.match(pwd));
+    }
+
+    String pwd = RandomStringUtils.randomAlphanumeric(8);
+    Assert.assertFalse(CommonlyUsedPwdChecker.match(pwd));
```

Comment:
This makes the tests flaky, you don't know when will `RandomStringUtils.randomAlphanumeric(8)` generate a string that contains the commonly used password fragments.

Reply:
Done.